ide/src/extension.ts

```diff
@@ -228,13 +228,18 @@ function buildCommands(): [string, (...args: any[]) => any][] {
         [
             "whycode.start",
             () => {
-                const document = vscode.window.activeTextEditor?.document;
-                if (document == undefined) {
-                    return;
+                let sessionPath = vscode.workspace.getConfiguration("whycode").get("sessionPath");
```

Comment:
I think this is fine as a first version, but I'd like to avoid this in the future, it seems brittle / counter-intuitive. Additionally, it won't work well once we make the server auto-start pre-existing proofs.